lib/config/definitions.js

```diff
@@ -1811,6 +1812,19 @@ const options = [
     env: false,
     mergeable: true,
   },
+  {
+    name: 'swift',
+    description: 'Configuration for Package.swift files',
+    stage: 'package',
+    type: 'object',
+    default: {
+      fileMatch: ['(^|/)Package\\.swift'],
+      versionScheme: 'swift',
+      rangeStrategy: 'bump',
```

Comment:
Why do you suggest `bump` by default?

Reply:
As I can see, `.from(<...>)` expressions mostly used with Swift, so `auto` strategy updates almost nothing in my examples. But maybe that's okay and should be reverted back to default.

Reply:
Also, current version of `getNewValue` support only `bump` semantics.

Reply:
Does `.from()` skip major versions as well? or is constrained to the current version?

And is there a lock file we could be bumping instead?

Reply:
It seems to be constrained on current major